thoth/glyph/cli.py

```diff
@@ -92,18 +93,19 @@ def generate(output: str) -> None:
 @click.option(
     "--message", "-m", type=str, required=True, help="Commit message to be classified",
 )
-@click.option("--model", type=str, help="Path to model")
+@click.option("--model", type=str, help="Type of classifer")
```

Comment:
I wonder how this will be user-friendly. It might be good to introduce a [choice here](https://click.palletsprojects.com/en/7.x/options/#choice-options) (names of the modeld). Which would be mapped to path internally. WDYT?

Reply:
An example of such usage, together with `Enum` can be found here:

https://github.com/thoth-station/adviser/blob/bed28ca39af2c6e4a34ef6f67b4179fc1a902257/thoth/adviser/cli.py#L307

Reply:
Fixed. Now using Enums.